### ISSUE TITLE
Fix Pathology Image Viewer in LabCAS - it just hangs on opening an image

### ISSUE THREAD

**Issue by hoodriverheather** [EDRN/labcas-ui#227]
## 🐛 Describe the Bug
@yuliujpl when trying to view an image in the BBD Data Collection, the viewer hangs.
    https://edrn-labcas.jpl.nasa.gov/labcas-ui/d/index.html?dataset_id=BBD_Pathology_Slide_Images/Anonymized_BBD_Slide_Images

There are also .scn images in the other dataset. Is it possible to open these with the pathology image viewer?
https://edrn-labcas.jpl.nasa.gov/labcas-ui/d/index.html?dataset_id=BBD_Pathology_Slide_Images/Source_BBD_Pathology_Slide_Images

Please update the image viewer.

**Comment by yuliujpl** [comment]
Yes, there were recent issues with the pathology viewer, I'm trying to see if I can swap it out for a different one!

**Comment by hoodriverheather** [comment]
@yuliujpl thanks. can you also take a look at this viewer in MCL? They have mentioned the path images we have in MCL PCA Pilot and when i click on an H&E image it kicks me back to the login screen.

https://mcl-labcas.jpl.nasa.gov/labcas-ui/f/index.html?file_id=PCA_Pilot_Data/H_and_E_Images/MDAnderson/AM00.svs
<img width="1452" alt="Screenshot 2024-09-25 at 5 23 40 PM" src="https://github.com/user-attachments/assets/69237f3d-45c6-4d78-ae01-c2d7081163a6">

**Comment by hoodriverheather** [comment]
@yuliujpl Any luck with this issue? I need to setup a call soon to review some EDRN data collections that are pathology images and will need the viewer to work. Here's a link to one of the collections - https://edrn-labcas.jpl.nasa.gov/labcas-ui/f/index.html?file_id=DCIS_Pathology_Slide_Images/Anonymized_DCIS_Slide_Images/3511005316.svs

Thanks! Heather

**Comment by yuliujpl** [comment]
I was able to go through the docker build process but ran out of space on the drive, requesting SAs for additional space!
________________________________
From: hoodriverheather ***@***.***>
Sent: Wednesday, November 13, 2024 3:47 PM
To: EDRN/labcas-ui ***@***.***>
Cc: Liu, David (US 3902-Affiliate) ***@***.***>; Mention ***@***.***>
Subject: [EXTERNAL] Re: [EDRN/labcas-ui] Fix Pathology Image Viewer in LabCAS - it just hangs on opening an image (Issue #227)


@yuliujpl<https://urldefense.us/v3/__https://github.com/yuliujpl__;!!PvBDto6Hs4WbVuu7!PqXery4r5WDhqlVnMQJjBY3QKl1M7kKyHldRqpqSPwlz2z6TQOJZ8l_4PGAjDhY77r5Nk_qs2HXaev0toRhQXpLqEAnFgA$> Any luck with this issue? I need to setup a call soon to review some EDRN data collections that are pathology images and will need the viewer to work. Here's a link to one of the collections - https://edrn-labcas.jpl.nasa.gov/labcas-ui/f/index.html?file_id=DCIS_Pathology_Slide_Images/Anonymized_DCIS_Slide_Images/3511005316.svs

Thanks! Heather

**Comment by hoodriverheather** [comment]
@yuliujpl Thanks for the update! I will wait to schedule a call until the viewer is up. :)

**Comment by yuliujpl** [comment]
Hi Heather,

Can you try again?

Thanks!
David
________________________________

**Comment by hoodriverheather** [comment]
@yuliujpl - still doesn't work. here's a link if you want to test - https://edrn-labcas.jpl.nasa.gov/labcas-ui/f/index.html?file_id=BBD_Pathology_Slide_Images/Anonymized_BBD_Slide_Images/3311000035.svs

**Comment by hoodriverheather** [comment]
@yuliujpl Any updates? This may also apply to MCL, however, EDRN is highest priority. Thanks, Heather

**Comment by yuliujpl** [comment]
Re-enabled, it's possible you might need incognito, though I tried to make it so you don't.

Thanks!
David
________________________________
From: hoodriverheather ***@***.***>
Sent: Thursday, November 21, 2024 3:24 PM
To: EDRN/labcas-ui ***@***.***>
Cc: Liu, David (US 3902-Affiliate) ***@***.***>; Mention ***@***.***>
Subject: [EXTERNAL] Re: [EDRN/labcas-ui] Fix Pathology Image Viewer in LabCAS - it just hangs on opening an image (Issue #227)


@yuliujpl<https://urldefense.us/v3/__https://github.com/yuliujpl__;!!PvBDto6Hs4WbVuu7!KEjb_AJt2LzGKsernfCZxgDuPJi3i4ZRV72mj3U9pcO8h1LQ3Oj9PItxvEwNVg-9rKV-v2zshI_E8XklI5limsk8xctiqA$> Any updates? This may also apply to MCL, however, EDRN is highest priority. Thanks, Heather

**Comment by hoodriverheather** [comment]
@yuliujpl That works great for EDRN. Can you also make that update for MCL LabCAS? Thanks!!
